packages/twenty-front/src/modules/favorites/scopes/FavoriteFoldersScope.tsx

```diff
@@ -0,0 +1,20 @@
+import { FavoriteFoldersScopeInternalContext } from '@/favorites/scopes/scope-internal-context/favoritesScopeInternalContext';
+import { ReactNode } from 'react';
+
+type FavoriteFoldersScopeProps = {
+  children: ReactNode;
+  favoriteFoldersScopeId: string;
+};
+
+export const FavoriteFoldersScope = ({
```

Comment:
not needed anymore in recoilcomonentV2 + not needed if we get rid of component states